pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -189,6 +193,14 @@ func (c *Reconciler) Reconcile(ctx context.Context, key string) error {
 			c.Recorder.Event(pr, corev1.EventTypeWarning, eventReasonFailed, "PipelineRun failed to update labels/annotations")
 			return err
 		}
+		updated = true
+	}
+
+	if updated {
+		metricsErr := c.metrics.RunningPRsCount(c.pipelineRunLister)
```

Comment:
nit: You don't need to call this `metricsErr` if you scope it to an `if` block:

`if err := c.metrics.RunningPRsCount(...); err != nil {`

Reply:
:+1: